sim/core/spell_effect.go

```diff
@@ -108,7 +108,13 @@ func (spellEffect *SpellEffect) PhysicalHitChance(unit *Unit) float64 {
 		hitRating += unit.PseudoStats.BonusRangedHitRating
 	}
 
-	return (hitRating / (MeleeHitRatingPerHitChance * 100)) - spellEffect.Target.HitSuppression
+	percent := (hitRating / (MeleeHitRatingPerHitChance * 100)) - spellEffect.Target.HitSuppression
```

Comment:
Min(0, ...)

Reply:
I thought about it but then remembered how it was slower than just the if check (something to do with inlining?)  I didn't even test perf of this and just did the if check hah.

Reply:
perf is minimally different... swapping to maxfloat